src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

Comment:
there could be a general class, one for each similar element

Reply:
fixed